tests/e2e/support/api/share/share.ts

```diff
@@ -0,0 +1,31 @@
+import { checkResponseStatus, request } from '../http'
+import { User } from '../../types'
+import join from 'join-path'
+export const createShare = async ({
+  user,
+  path,
+  shareWith,
+  shareType,
+  role
+}: {
+  user: User
+  path: string
+  shareWith: string
+  shareType: string
+  role: string
+}): Promise<void> => {
+  const shareTypeToIntString = shareType === 'group' ? '1' : shareType === 'public link' ? '3' : '0'
```

Comment:
Best to store shareType object somewhere and use to get the respective type number
```suggestion
// somewhere
export const shareTypes = {
  user: '0',
  group: '1',
  public: '3'
}
```